File: src/taburet/transactions/_design/transactions/views/balance_for_date/reduce.js
function(keys, values, rereduce) {
	var debet = 0
	var kredit = 0
	values.forEach( function(v) {
		debet += v.debet
		kredit += v.kredit
	})
	return {debet:Math.round(debet * 100.0) / 100.0,
		kredit:Math.round(kredit * 100.0) / 100.0}
}

File: src/taburet/transactions/_design/transactions/views/balance_for_account/reduce.erl
fun(Keys, Values, ReReduce) ->
	Debet = lists:sum([D || {[{<<"debet">>, D}, {<<"kredit">>, K}]} <- Values ]),
	Kredit = lists:sum([K || {[{<<"debet">>, D}, {<<"kredit">>, K}]} <- Values ]),
	{[{debet, erlang:round(Debet*100.0)/100.0}, {kredit, erlang:round(Kredit*100.0)/100.0}]}
end.
